### DISCUSSION TITLE
Welcome to hedera-accelerator-stablecoin Discussions!

### DISCUSSION THREAD

**Discussion by gregscullard** [hashgraph/stablecoin-studio#23] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by fredaspen1** [comment]
@gregscullard how to set up a stablecoin and its transaction fees in another currencies than USD, such as Euro? On the stablecoin studio it seems like it's only possible to build stablecoin backed by USD. Please explain me
